crypto/engine/eng_cryptodev.c

```diff
@@ -184,7 +186,6 @@ static struct {
     {
         CRYPTO_AES_CBC, NID_aes_256_cbc, 16, 32,
     },
-# ifdef CRYPTO_AES_CTR
```

Comment:
The guard didn't work, CRYPTO_AES_CTR is enum value and if it's defined cryptodev breaks. The code inside (after small fix) appears to work well.